hs_core/receivers.py

```diff
@@ -1,6 +1,8 @@
+"""Signal receivers for the hs_core app.
 
-# Note: this module has been imported in the models.py in order to receive signals
-# se the end of the models.py for the import of this module
+Note: this module has been imported in the models.py in order to receive signals
+see the end of the models.py for the import of this module
+"""
```

Comment:
@aphelionz I believe this comment does not apply any more. The import of this module has been moved to HSCoreAppConfig class instead.